cmd/gobrew/main.go

```diff
@@ -124,17 +150,22 @@ func main() {
 	}
 }
 
+// isArgAllowed checks if the arg is allowed
+// but ignored flags
+//
+// if the any arg is not allowed, it will return false
 func isArgAllowed() bool {
 	ok := true
-	if len(os.Args) > 1 {
-		_, ok = Find(allowedArgs, os.Args[1])
-		if !ok {
-			return false
+	for i := range os.Args {
```

Comment:
Another thing, after switching to using a loop for checking, there is a problem with the command sequence. And such a sequence will be correct:
```sh
go run ./cmd/gobrew latest use
```

I'll fix it